--- app/build.gradle
+++ app/build.gradle
@@ -62,7 +62,7 @@
     def fileFilter = ['**/R.class', '**/R$*.class', '**/BuildConfig.*', '**/Manifest*.*', '**/*Test*.*', 'android/**/*.*']
     def debugTree = fileTree(dir: "${buildDir}/tmp/kotlin-classes/debug", excludes: fileFilter)
     def mainSrc = "${project.projectDir}/src/main/kotlin"
-
+    println "Coverage Report Debug Tree Size: " + debugTree.files.size()
     sourceDirectories.setFrom(files([mainSrc]))
     classDirectories.setFrom(files([debugTree]))
     executionData.setFrom(fileTree(dir: "${project.buildDir}", includes: [
